flasc/model_fitting/yaw_pow_fitting.py

```diff
@@ -117,6 +143,20 @@ def estimate_cos_pp_fit(
         opt_pp_range=(1.0, 10.0),
         opt_Ns=41,
     ):
+        """Estimate the best fit for a cos(x-x0)^pp curve.
+
+        Args:
+            opt_yshift_range (tuple): Tuple containing the lower and upper bounds for the y shift.
+                Default is None.
+            opt_bias_range (tuple): Tuple containing the lower and upper bounds for the bias.
+                Default is (-15.0, 15.0).
+            opt_pp_range (tuple): Tuple containing the lower and upper bounds for the power.
+                Default is (1.0, 10.0).
+            opt_Ns (int): Number of samples to use for the optimization. Default is 41.
+
+        Returns:
+            x_opt (np.array): Optimal parameters for the cos(x-x0)^pp curve.
```

Comment:
Here we should probably describe what the elements of `x_opt` are... presumably, it contains both the optimal `x0` and the optimal `pp` for the fit? What order are they in?

Reply:
Ok good idea